include/nv/target

```diff
@@ -26,6 +26,8 @@
 #  define _NV_BITSET_ATTRIBUTE
 #endif
 
+#if __cplusplus >= 201103L
```

Comment:
I would like to see instead:
```
#if c++11
# define _NV_CONSTEXPR_11
#endif
...

#if _NV_CONSTEXPR_11
... <- nv::target:: stuff
#endif
```

But I can address this myself later. I won't hold up the commit on this.

Reply:
`constexpr` isn't the only C++11 feature in there.  There's also an `enum class`.  So I'm a little reluctant to use a macro that focuses on the `constexpr`.